src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -1831,6 +1865,12 @@ public async Task<JObject> CreateJObjectForValueType(MonoBinaryReader retDebugge
             byte[] valueTypeBuffer = new byte[endPos - initialPos];
             retDebuggerCmdReader.Read(valueTypeBuffer, 0, (int)(endPos - initialPos));
             retDebuggerCmdReader.BaseStream.Position = endPos;
+            var valueTypeFields = JObject.FromObject(new
+            {
+                result = valueTypeFieldsPublic,
```

Comment:
Can you explain what we return, based on the various "inputs" - `accessorPropertiesOnly`, `sortByAccessLevel`? When do we include fields, properties, and which public/private/internal, and inherited ones or not. Not just here, but in general.